scripts/build/TestImpactAnalysis/tiaf_driver.py

```diff
@@ -62,4 +62,5 @@ def test_failure_policy(value):
         sys.exit(0)
     except:
         # Non-gating will be removed from this script and handled at the job level in SPEC-7413
-        sys.exit(0)
\ No newline at end of file
+        print("Exception caught by TIAF driver")
+        sys.exit(0)
```

Comment:
The `sys.exit(0)` should be place outside of the exception block.

Also globally catching an exception is a bad practice.
Applications should handle the exceptions it expects, and not the ones that it doesn't it.
Furthermore it hides real errors as the exception is being silenced by the process.

Reply:
I think @jonawals intention here is to never fail the script because this should never produce a build failure

Reply:
Discussed offline, short term solution is to log the exception, long term solution is to properly handle expected abnormal behavior when the script is ready to gate in the next dev phase.